war/src/main/js/plugin-manager-ui.js

```diff
@@ -63,6 +63,8 @@ document.addEventListener("DOMContentLoaded", function () {
         filterInput.parentElement.classList.add("jenkins-search--loading");
     });
 
+    filterInput.focus();
```

Comment:
all the other pages autofocused this input on load so I made that consistent at the same time. (this fix alone would have addressed 'mouse-moving' concerns :)